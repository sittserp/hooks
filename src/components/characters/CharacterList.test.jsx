import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterList from './CharacterList';

describe('CharacterList component', () => {
  afterEach(() => cleanup());
  it('renders CharacterList', () => {
    const { asFragment } = render(
      <CharacterList characters={[]} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
