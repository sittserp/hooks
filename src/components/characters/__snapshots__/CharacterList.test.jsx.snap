// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterList component renders CharacterList 1`] = `
<DocumentFragment>
  <ul
    data-testid="CharacterList"
  />
</DocumentFragment>
`;
